features/authentication.feature

```diff
@@ -117,6 +117,25 @@ Feature: OAuth authentication
     Then the output should not contain "github.com password for mislav"
     And the file "../home/.config/hub" should contain "oauth_token: OTOKEN"
 
+  Scenario: Credentials from GITHUB_TOKEN
+    Given the GitHub API server:
+      """
+      post('/authorizations') {
+        assert_basic_auth 'mislav', 'kitty'
+        json :token => 'OTOKEN'
+      }
+      get('/user') {
+        json :login => 'mislav'
+      }
+      post('/user/repos') {
```

Comment:
How might I go about doing that? `params["token"]`? Is there a `request.headers["Authorization"]` variable?